content/blog/posts/o3de-ux-past-present-future.md

```diff
@@ -0,0 +1,107 @@
+---
+title: "Explore the Past, Present, & Future of O3DE's First-time User Experience"
+description: "Learn about how Amazon Web Services developed the first-time user experience for Open 3D Engine (O3DE) from Bhanuja Sanghavi, one of the UX designers contributing to O3DE."
+date: 2022-01-14
+slug: o3de-ftue-blog-post-1
+author: Bhanuja Sanghavi, Amazon Web Services
+blog_img: "/images/blog/announcement_thumbnail.jpg"
+full_img: "/images/blog/bjds-logo.jpg"
+---
+
+Learn about how Amazon Web Services developed the first-time user experience for Open 3D Engine (O3DE) from Bhanuja Sanghavi, one of the UX designers contributing to O3DE.
+
```

Comment:
Can we add some line breaks here. as the explore the past, present and future heading is the main heading. I would like to give it some white space before the tag line above.